app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copi_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] == "cp" and command_parts[1] != command_parts[2]:
```

Comment:
Firstly, you should check that `command_parts` contains 3 elements
```suggestion
    if len(command_parts) == 3 and command_parts[0] == "cp" and command_parts[1] != command_parts[2]:
```